demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -548,7 +600,13 @@ def build_rn_desc(self, _type: FileType = None, content_name: str = '', desc: st
                 if self.update_type == 'documentation':
                     rn_desc += '- Documentation and metadata improvements.\n'
                 else:
-                    rn_desc += f'- {text or "%%UPDATE_RN%%"}\n'
+                    deprecate_rn = ''
+                    if _type in (FileType.INTEGRATION, FileType.SCRIPT, FileType.PLAYBOOK):
+                        deprecate_rn = get_deprecated_rn(path, _type)
+                    if deprecate_rn:
+                        rn_desc += f'- {text}\n{deprecate_rn}'
```

Comment:
if text isnt provided we'll have an extra blank line here. Lets add
if text:
    rn_desc += f'{text}\n'
rn_desc +=deprecate_rn